Add support for environment variable based rack awareness (#12593)
strimzi/strimzi-kafka-operator: documentation/api/io.strimzi.api.kafka.model.kafka.KafkaClusterSpec.adoc

** `max.connection.creation.rate`

[id='property-kafka-brokerRackInitImage-{context}']
= Configuring topology-label-based rack awareness and init container images

Rack awareness is enabled using the `rack` property.

> When topology-label-based rack awareness is enabled, Kafka broker pods use init container to collect the labels from the Kubernetes cluster nodes.

Reviewer: ```suggestion
When topology-label-based rack awareness is enabled, Kafka broker pods use an init container to collect labels from Kubernetes nodes.
```
